cmd/brew-setup-nginx-conf

```diff
@@ -79,6 +79,15 @@ if `readlink /etc/resolver 2>/dev/null`.chomp != "/usr/local/etc/resolver"
   end
 end
 
+if File.exist? "/etc/pf.anchors/dev.strap"
+  unless system "sudo -n true >/dev/null"
+    puts "Asking for your password to setup launch_socket_server:"
+  end
+  system "sudo rm /etc/pf.anchors/dev.strap"
```

Comment:
Also want to delete `/etc/pf.anchors/dev.github` and remove `dev.strap` references from `/etc/pf.conf` e.g. with `grep -v "dev.strap" /etc/pf.conf` written to a temporary file.